virttest/remote_commander/remote_master.py

```diff
@@ -259,6 +265,15 @@ def listen_errors(self, cmd):
                             remote_interface.MessengerError)):
             raise cmd
 
+    def listen_queries(self, cmd):
+        """Manage queries from slave side.
+        """
```

Comment:
Yeah, it was written to it, but it doesn't mean it's the only way... My point is that if it works fine, than it's good. But once you get some failures, it'll be pain debugging it.